src/main/java/frc/robot/Ports.java

```diff
@@ -10,8 +12,13 @@
         public static final boolean IS_AUX_SENSOR_INVERTED = false;
     }
 
+    public static class Shooter {
+        public static final int MOTOR = 0; // Main motor port.
+        public static final TalonFXConfiguration CONFIGURATION = new TalonFXConfiguration();
```

Comment:
If you don't use this configuration why do use it? also, it should be in Constants

Reply:
I use it in the shooter constructor.